ChatQnA/docker_compose/intel/cpu/xeon/compose.perf.yaml

```diff
@@ -0,0 +1,7 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
If you want to test this yaml in CI, you can have a test scripts to apply this yaml, refer to this, https://github.com/opea-project/GenAIExamples/blob/7072d2cb87a5481e7533c8cb1f9fb63394098446/ChatQnA/tests/test_compose_on_xeon.sh, 
mainly change this part `docker compose -f compose.yaml -f compose.telemetry.yaml up -d --quiet-pull` to `docker compose -f compose.yaml -f compose.perf.yaml up -d --quiet-pull`, with minimal test.